src/main/java/com/atlassian/db/replica/api/state/State.java

```diff
@@ -14,6 +14,7 @@ private State(String name) {
     public static final State MAIN = new State("MAIN");
     public static final State REPLICA = new State("REPLICA");
     public static final State CLOSED = new State("CLOSED");
+    public static final State COMMITED_MAIN = new State("COMMITED_MAIN");
```

Comment:
Do we need to expose those `public static final State COMMITED_MAIN` and its siblings?

Reply:
I guess it's easier to compare classes than.
Currently, I'm working on a query level visibility, and I'm not sure if it will not deprecate the current STATE level visibility changes. I'll postpone the merge and see how it evolves.

Reply:
I'll merge it and resolve this issue before 1.0.0 release.